openmls/src/group/errors.rs

```diff
@@ -206,3 +211,31 @@ implement_error! {
         }
     }
 }
+
+implement_error! {
+    pub enum FramingValidationError {
+        WrongGroupId = "Message group ID differs from the group's group ID.",
+        WrongEpoch = "Message epoch differs from the group's epoch.",
+        UnknownSender = "The sender is not a member of the group.",
```

Comment:
If I understand the code correctly, this is thrown either if the advertised sender is outside of the tree or if it has a public key that can't be found in the tree. It would be nice to address these cases separately here, since one will go away with preconfigured senders or external init and one will always be a straight-up error.

Reply:
Same for the error below, I think.